Processing/BRCA_Deep_Dive.Rmd

```diff
@@ -2392,7 +2392,195 @@ kable(Well_Capacity)
+
+This 2023 version of WHP was created from updated national wildfire hazard datasets of annual burn probability and fire intensity generated by the USDA Forest Service, Rocky Mountain Research Station with the large fire simulation system (FSim). Vegetation and wildland fuels data from LANDFIRE 2020 (version 2.2.0) were the primary inputs to the updated FSim modeling work and therefore form the foundation for this version of the WHP. As such, the data presented here reflect landscape conditions as of the end of 2020. LANDFIRE 2020 vegetation and fuels data were also used directly in the WHP mapping process, along with updated point locations of fire occurrence ca. 1992-2020. With these datasets as inputs, we produced an index of WHP for all of the conterminous United States at 270-meter resolution. We present the final WHP map in two forms: 1) continuous integer values, and 2) five WHP classes of very low, low, moderate, high, and very high. On its own, WHP is not an explicit map of wildfire threat or risk, but when paired with spatial data depicting highly valued resources and assets such as structures or powerlines, it can approximate relative wildfire risk to those specific resources and assets. WHP is also not a forecast or wildfire outlook for any particular season, as it does not include any information on current or forecasted weather or fuel moisture conditions. It is instead intended for long-term strategic fuels management.
+
+```{r}
+# GRAB MEAN WATER SUPPLY RANKING OF WHP IN THE WATER SUPPLY PYROME:
+  
+# continuous fire hazard potential, cropped to water supply pyrome:
+whp_cnt <- terra::rast("data/all/usfs_fire/whp2023_GeoTIF/whp2023_cnt_conus.tif") %>%
+  terra::crop(., sf::st_transform(water_supply_pyromes, 5070), 
+              mask = TRUE, touches = TRUE) %>%
+  na.omit()
+  
+# transform the ranked values into percentiles (at 1 percent intervals):
+
+values <- terra::values(whp_cnt, na.rm = TRUE)
+
+quantiles <- quantile(values, probs = seq(0, 1, by = 0.01)) %>%
+  tibble::as_tibble() %>%
+  dplyr::mutate(to = c(lead(value, 1))) %>%
+  dplyr::mutate(to = ifelse(is.na(to), max(values) + 1, to)) %>%
+  dplyr::rename(from = value) %>%
+  dplyr::mutate(becomes = seq(0, 1, by = 0.01))
+
+# Return the percentiles based on 1% threshold defined above
+whp_quantiled <- terra::classify(whp_cnt, rcl = quantiles, right = TRUE)
+
+# plot(whp_quantiled)
+
+# grab percentiles within the water supply watershed:
+water_supply_fire_potential <- whp_quantiled %>%
+  crop(sf::st_transform(watersupply_watershed, 5070), mask = TRUE) 
+
+# calculate area-weightem mean of the water supply watershed:
+mean_fire_potential <- terra::zonal(water_supply_fire_potential, 
```

Comment:
So, we are essentially reporting the water supply's wildfire hazard potential in terms of its rank within the entire pyrome that the water supply lies in? Is this the most relevant comparison?

Reply:
I think it is, if we want to go the "continuous" values route, since the area is still quite large and pyromes are a prevalent analysis level. Plus it is the area used in Gao. But! As discussed in chat, we may now be more interested in looking at the classified version of the dataset which gets rid of the need to group/compare at all. I've added code in to explore both options... we now have the percentage of watershed classified as "high" wildfire hazard potential.